### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by katiyelae**
# Pull Request Template

## Description

Please provide a brief description of the changes introduced in this pull request. Explain why these changes are necessary and how they address the issue or task. Include any relevant context or background information.

## Checklist

- [X] I have tested these changes locally to ensure they work as expected.
- [X] I have reviewed and addressed any code comments, suggestions, or feedback.
- [x] I have updated the project documentation if these changes affect it.
- [ ] I have resolved any merge conflicts (if applicable).
- [ ] I have run automated tests, and they pass without errors.
- [ ] I have squashed my commits into a single, coherent commit (if necessary).

## Related Issues

Closes #[Issue Number]

If your pull request resolves a specific issue, reference it here using the `Closes #` syntax.

## Screenshots (if applicable)

Include screenshots or GIFs that demonstrate the changes visually, especially for UI-related changes.

## Additional Notes

It's Alive! Things are beginning to work together thanks to getting the React Router Paths set up. Check out the Home Page, Resources Page, and the Questions Page. The navbar is also set up on each page. Still trying to figure out how to get the styling to link and show, but hopefully can get that figured out this week. 

## Reviewers

- @funsav3r
- @IrascibleMidnight
- @juhi1103
- @ilgincevik
- @tihardy
- @erinaperez
- @ann-codes 
- @contributor8

Assign the PR to relevant reviewers from the contributor's list. Reviewers, please ensure that the changes meet project standards, follow coding guidelines, and fulfill the project's requirements.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,18 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+import NavBar from './components/NavBar';
+import RoutePaths from './components/RoutePaths'
+// import Home from './routes/Home'
```
Comment (ann-codes) [inline src/App.jsx:4]: this can just be deleted

File: src/index.css [hunk 2]
```diff
@@ -1,74 +1,172 @@
+/* this file contains global CSS styles that apply to the entire application */
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+.landing-page .overlap {
```
Comment (ann-codes) [inline src/index.css]: Optional - If the landing page is Home.jsx, this should ideally go along with the Home.css

File: src/routes/Questions.jsx [hunk 3]
```diff
@@ -1 +1,5 @@
-import React from 'react';
```
Comment (ann-codes) [inline src/routes/Questions.jsx:1]: I think you still need to import react?

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    list-style-type: none; 
+    margin: 0; padding: 0; 
+    overflow: hidden; 
+    background-color: unset ; 
+} 
+ul.topnav li {
+    float: left;
+}
+
+.landing-page .overlap {
```
Comment (ann-codes) [inline src/style.css]: seems to be a dupe of the index.css?

**Review by ann-codes — APPROVED** [review]

File: src/components/NavBar.jsx
Comment (erinaperez) [inline src/components/NavBar.jsx:1]: Looks great Kati! Should we double check whether we want a nav link to go to the questions page? I know "Get started" takes the user to questions, but might be helpful to have it in nav as well for accessibility?

File: src/components/RoutePaths.jsx
Comment (erinaperez) [inline src/components/RoutePaths.jsx:1]: I haven't seen the routes broken into their own component like this before - makes sense and looks good.

**Review by erinaperez — APPROVED** [review]

**Pull request merged**